cms/djangoapps/contentstore/tests/test_course_listing.py

```diff
@@ -45,11 +45,14 @@ def setUp(self):
         self.user = UserFactory()
         self.factory = RequestFactory()
         self.request = self.factory.get('/course')
+
+        # pylint: disable=protected-access
+        self.split_store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.split)
```

Comment:
`self.store.make_usage_key(org, course, run)` was returning `mongo` course locator and we were creating `split` courses which were causing conflicts. 

[default_modulestore](https://github.com/edx/edx-platform/blob/93656e39eea894273b5737442d65e29df4ce9cdd/common/lib/xmodule/xmodule/modulestore/mixed.py#L314)

I am now using `CourseLocator`